server/src/schemas/donatedItems.ts

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi';
+//DonatedItem schema
+export const donatedItemSchema = Joi.object({
+    itemType: Joi.string().required(),
+    currentStatus: Joi.string().valid('Received', 'Pending', 'Processed', 'Delivered').required(),
+    donorEmail: Joi.string().email().required(),
+    program: Joi.string().required(),
+    dateDonated: Joi.date().required(), // Validates as a proper date
+});
+
+// DonatedItemStatus schema
+export const donatedItemStatusSchema = Joi.object({
+    statusType: Joi.string().valid('Received', 'Pending', 'Processed', 'Delivered').required(),
```

Comment:
I think schema checks for basic validation like types and required fields, my suggestion is we can also improve validation for specific fields, such as the length of strings 
(note: in the front end, the user is given the option to fill out the form but not any textual input. so the validations or the checks might not be necessary in that case.)